R/Learner.R

```diff
@@ -164,6 +164,10 @@ Learner = R6Class("Learner",
       self$predict_types = assert_ordered_set(predict_types, names(mlr_reflections$learner_predict_types[[task_type]]),
         empty.ok = FALSE, .var.name = "predict_types")
       private$.predict_type = predict_types[1L]
+
+      if (!is.null(private$.bundle) && !is.null(private$.unbundle)) {
```

Comment:
I would keep Michel's style and set "bundle" as a property in the initialize of the learner.

Reply:
This case is different because we can infer the property because it is something that is implemented in `mlr3` and not in the upstream package. I don't really mind but I don't see the benefit of adding a possible source of error.